src/main/java/frc/robot/RobotContainer.java

```diff
@@ -101,6 +101,11 @@ public void initialize() {
     // Put the chooser on the dashboard
     SmartDashboard.putData(autonChooser);
 
+    // Put the buttons for zeroing the mechanisms on the dashboard
+    SmartDashboard.putData("Zero Intake Based on Current Pos", new InstantCommand(intake::zeroIntakeAtCurrentPos, intake));
+    SmartDashboard.putData("Zero Lift Based on Current Pos", new InstantCommand(lift::zeroLiftAtCurrentPos, lift));
+    SmartDashboard.putData("Zero Swerve Based on Current Pos", new InstantCommand(drive::zeroSwerveAtCurrentPos, drive));
```

Comment:
These commands need `.ignoringDisable(true)` so that we can run them while the robot is not enabled